tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs

```diff
@@ -6,23 +6,39 @@ namespace Nox.Types.Tests.EntityFrameworkTests;
 public abstract class TestWithSqlite : IDisposable
 {
     private const string InMemoryConnectionString = "DataSource=:memory:";
+    //private const string InMemoryConnectionString = @"DataSource=test.db";
```

Comment:
why add this?

Reply:
just for debugging proposes, I will remove